source/usage-examples/insert-operations.txt

```diff
@@ -2,7 +2,8 @@
 Insert Operations
 =================
 
-.. default-domain:: mongodb
+.. meta::
+   :description: See examples of inserting data into MongoDB by using the {+driver-long+}.
```

Comment:
Suggestion: for the usage examples, does this sound a little more natural?

```suggestion
   :description: See how to insert data into MongoDB by using the {+driver-long+}.
```

Reply:
By natural, did you mean avoiding using "examples" when the title of the page might include "examples"?
I don't have a strong preference for either your suggestion or the original text.

Reply:
I think including the word example is unnecessary-- including the word "how" might make it easier for users to pick up the point of these pages

Reply:
Makes sense. I think maybe I'll update this and all similar instances to "Learn how" since I think that seems more frequently used in our style guide examples.

Reply:
On second thought, the title of the page and how it would show up in search results is "Insert Operations", not "Usage Examples", so I think "examples" is important to mention. I will try rephrasing it a bit to avoid the gerund.